### PR TITLE
[color-hash] Allow more than three values to be set for lightness and saturation

### PR DESCRIPTION
**Pull request by ryota-abe**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by JohannesHoppe** [comment]
This looks good. But I won't check it more closely, because I don't use color-hash in any project anymore and I don't have resources to maintain this privately.

File: types/color-hash/index.d.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ interface HueObject {
 }
 
 type Hue = number | HueObject | ReadonlyArray<HueObject>;
-type Lightness = number | ColorValueArray;
-type Saturation = number | ColorValueArray;
+type Lightness = number | ReadonlyArray<number>;
+type Saturation = number | ReadonlyArray<number>;
```
Comment (peterblazejewicz) [inline types/color-hash/index.d.ts:16]:
```suggestion
type Lightness = number | number[];
type Saturation = number | number[];
```
those are props of the options passsed to constructor, not args, not parameters, why readonly?

Comment (ryota-abe) [reply types/color-hash/index.d.ts]: You are absolutely right. Thank you for pointing this out.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
checked against source

**Comment by ryota-abe** [comment]
Ready to merge

**Pull request merged**